docs/source/conf.py

```diff
@@ -34,3 +34,4 @@
 intersphinx_mapping = {"https://docs.python.org/": None}
 
 autoclass_content = "both"
+master_doc = "index"
```

Comment:
I needed to add this for configly for RTD to build, maybe we just haven't built docs in a while, but wanted to preempt that from failing.